parseDataset.js

```diff
@@ -123,6 +123,14 @@ lines.slice(TITLE_LINE + 1).forEach((line) => {
       currentState.ingredient_id + "." + lineObject.quantity_unit_id;
     lineObject.quantity_id = quantity_id;
 
+    // Defaultize values
+    lineObject.quantity_unit_id = lineObject.quantity_unit_id || "g";
+
+    // quantity_unit_id can contain a prefix that indicates the type of ingredient
+    // e.g. thigh-unit, meat-g, ground-meat-g
+    // extract the actual unit suffix
+    lineObject.quantity_unit = lineObject.quantity_unit_id.replace(/^.*-/, '');
```

Comment:
Ok, it's just weird because you mix the id with some data.

For me ids are just for identification purposes, they can be a hash string with no meaning, and you need to add columns for additional data. For example:

For example `beef.ground` and `beef.piece` could be the two ids, and have a property `unit` which can get `'g'`, `'kg'`, `'ml'`, ...

For example, you replaced "water.ml" with "water.cl" I think the id should be kept, the same (could be `water.`) and the unit should be a property Otherwise the program will end-up with messages such that "water.ml" not found

Reply:
Ok, sounds good, I will add a separate column for the quantity unit.

Reply:
@alexfauquette does this look better?